Error occurred at /people
Port: 5173

Stack trace:
TypeError: goToMonth is not a function
    at Object.handleMonthChange [as current] (src/components/ui/calendar.tsx:37:7)
    at Object.onValueChange (node_modules/.vite/deps/chunk-2OYJTKPX.js?v=021026f1:67:60)
    at handleSelect (node_modules/.vite/deps/@radix-ui_react-select.js?v=9e824666:866:17)
    at node_modules/.vite/deps/@radix-ui_react-select.js?v=9e824666:912:59
    at handleEvent (node_modules/.vite/deps/chunk-2RDUA72F.js?v=021026f1:7:49)
    at HTMLUnknownElement.callCallback2 (node_modules/.vite/deps/chunk-WRD5HZVH.js?v=021026f1:3674:22)
    at Object.invokeGuardedCallbackDev (node_modules/.vite/deps/chunk-WRD5HZVH.js?v=021026f1:3699:24)
    at invokeGuardedCallback (node_modules/.vite/deps/chunk-WRD5HZVH.js?v=021026f1:3733:39)
    at invokeGuardedCallbackAndCatchFirstError (node_modules/.vite/deps/chunk-WRD5HZVH.js?v=021026f1:3736:33)
    at executeDispatch (node_modules/.vite/deps/chunk-WRD5HZVH.js?v=021026f1:7014:11)
